qa/evil-tests/src/test/java/org/elasticsearch/bootstrap/BootstrapCliParserTests.java

```diff
@@ -220,10 +229,11 @@ public void testHelpWorks() throws Exception {
         tuples.add(new Tuple<>("-h", "elasticsearch.help"));
 
         for (Tuple<String, String> tuple : tuples) {
-            terminal = new CaptureOutputTerminal();
+            terminal.resetOutput();
             BootstrapCLIParser parser = new BootstrapCLIParser(terminal);
             ExitStatus status = parser.execute(args(tuple.v1()));
             assertStatus(status, OK_AND_EXIT);
+            // nocommit
```

Comment:
Oops yes, it was already fixed, the nocommit was with this call commented out. I hadn't caught it yet because my async test run had I had started failed for unrelated reasons, before getting to evil-tests.